app/src/main/java/travelbook/android/app/activity/MemoriesActivity.java

```diff
@@ -24,5 +25,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         final List<Memory> memories = memoriesDao.fetchAll();
         Log.d(TAG, "Fetched memories: " + memories);
+
+        TravelBookDatabaseOpenHelper travelBookDatabaseOpenHelper = new TravelBookDatabaseOpenHelper(this);
```

Comment:
Revert MemoriesActivity

Reply:
what do you mean "revert" ?

Reply:
I mean the MempriesActivity should not be included in the pull request.